Assets/HoloToolkit-Examples/SpectatorView/Scripts/SpectatorViewNetworkManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿using HoloToolkit.Unity.Preview.SpectatorView;
```

Comment:
@davidkline-ms Does sample code require the MIT license header?

Reply:
For consistency with ColorChanger.cs: Adding license header.

Thanks Tim!